### PR TITLE
Add height for Readstate api response 

### PR DESCRIPTION
**Pull request by koseoyoung**
works on #2258

### CODE REVIEW

File: action/protocol/staking/protocol.go [hunk 1]
```diff
@@ -376,10 +376,18 @@ func (p *Protocol) ReadState(ctx context.Context, sr protocol.StateReader, metho
 		err = errors.New("corresponding method isn't found")
 	}
 	if err != nil {
-		return nil, err
+		return nil, uint64(0), err
+	}
+	bytes, err := proto.Marshal(resp)
```
Comment (yutongp) [inline action/protocol/staking/protocol.go:381]: nit: `bytes` -> `data`. (`bytes` is a little bit similar to go type `byte`)

File: api/api.go [hunk 2]
```diff
@@ -448,12 +448,15 @@ func (api *Server) ReadState(ctx context.Context, in *iotexapi.ReadStateRequest)
...
 	}
-	data, err := api.readState(ctx, p, in.GetHeight(), in.MethodName, in.Arguments...)
+	data, readStateHeight, err := api.readState(ctx, p, in.GetHeight(), in.MethodName, in.Arguments...)
 	if err != nil {
 		return nil, status.Error(codes.NotFound, err.Error())
 	}
 	out := iotexapi.ReadStateResponse{
 		Data: data,
+		BlockIdentifier: &iotextypes.BlockIdentifier{
+			Height: readStateHeight,
```
[the remark this answers is not in the document]
Comment (koseoyoung) [reply api/api.go:458]: fixed in latest commit

File: action/protocol/staking/protocol.go (hunk 1 above)
Comment (koseoyoung) [reply action/protocol/staking/protocol.go:381]: fixed in latest commit

Comment (koseoyoung) [reply action/protocol/staking/protocol.go]: addressed on latest commit

File: api/api.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (koseoyoung) [reply api/api.go:465]: addressed on latest commit

File: go.sum [hunk 3]
```diff
@@ -495,6 +495,8 @@ github.com/xordataexchange/crypt v0.0.3-0.20170626215501-b2862e3d0a77/go.mod h1:
 github.com/zenazn/goji v0.9.0/go.mod h1:7S9M489iMyHBNxwZnk9/EHS098H4/F6TATF2mIxtB1Q=
 go.etcd.io/bbolt v1.3.2 h1:Z/90sZLPOeCy2PwprqkFa25PdkusRzaj9P8zm/KNyvk=
 go.etcd.io/bbolt v1.3.2/go.mod h1:IbVyRI1SCnLcuJnV2u8VeU0CEYM7e686BmAb1XKL+uU=
+go.etcd.io/bbolt v1.3.5 h1:XAzx9gjCb0Rxj7EoqcClPD1d5ZBxZJk0jbuoPHenBt0=
```
[the remark this answers is not in the document]
Comment (koseoyoung) [reply go.sum:498]: I tried this command, but still the same.

[the remark this answers is not in the document]
Comment (yutongp) [reply go.sum:498]: ya, it is from iotex-election. cannot be removed from iotex-core

**Pull request merged**